internal/cmd/downloadcmds/forge.go

```diff
@@ -0,0 +1,37 @@
+package downloadcmds
+
+import (
+	"strings"
+
+	"github.com/talwat/pap/internal/fs"
+	"github.com/talwat/pap/internal/global"
+	"github.com/talwat/pap/internal/jarfiles"
+	"github.com/talwat/pap/internal/jarfiles/forge"
+	"github.com/talwat/pap/internal/log"
+	"github.com/talwat/pap/internal/net"
+	"github.com/urfave/cli/v2"
+)
+
+func DownloadForgeCommand(cCtx *cli.Context) error {
+	url := forge.GetURL(
+		global.MinecraftVersionInput,
+		global.ForgeInstallerVersion,
+		global.ForgeUseLatestInstaller,
+	)
+	surl := strings.Split(url, "/")
+
+	net.Download(
+		url,
+		"resolved forge-installer jarfile not found",
+		surl[len(surl)-1],
```

Comment:
Maybe instead just make it something like `forge.jar` or `forge-installer.jar`.

Reply:
After running the installer, it creates a file which ends in `forge.jar`. I'd rather keep the `-installer` suffix to avoid confusion

Reply:
I mean instead of using the last part of the url, just use `forge-installer.jar`

Reply:
Can we meet in the middle and agree on a naming scheme such as
`forge-1.19-latest-installer.jar`? Could cause conflicts down the road if every file is named the same thing.